src/ui.py

```diff
@@ -18,25 +18,7 @@ def enable_data_dependent_buttons(self):
     for item in self.datadict.values():
         if item.selected and isinstance(item, Item):
             enabled = True
-    win = self.main_window
-    dependent_buttons = [
-        win.shift_vertically_button,
-        win.translate_x_button,
-        win.translate_y_button,
-        win.multiply_x_button,
-        win.multiply_y_button,
-        win.smooth_button,
-        win.fourier_button,
-        win.inverse_fourier_button,
-        win.normalize_button,
-        win.center_button,
-        win.derivative_button,
-        win.integral_button,
-        win.transform_button,
-        win.combine_button,
-    ]
-    for button in dependent_buttons:
-        button.set_sensitive(enabled)
+    self.main_window.shift_vertically_button.set_sensitive(enabled)
```

Comment:
If we bind the buttons to  item_list.visible, we can get rid of this entire function

Reply:
No, because when no item is selected, the buttons become insensitive too, whilst the item_list isn't invisible

Reply:
Yep, didn't consider the  other condition of having no item selected. My bad!

Connecting them to the shift_vertically function removes the necessity to assign them in `window.py.` Otherwise I don't have a strong preference to either solution, think this is fine.